ivadomed/config/config_bids.json

```diff
@@ -99,7 +103,7 @@
         },
         {
             "name": "datatype",
-            "pattern": "[/\\\\]+(anat|beh|dwi|eeg|fmap|func|ieeg|meg|perf|micr|ct)[/\\\\]+"
+            "pattern": "[/\\\\]+(anat|beh|dwi|eeg|fmap|func|ieeg|meg|micr|perf|pet|ct)[/\\\\]+"
```

Comment:
Note to the reviewer:
This file was use as a "custom" pybids config to index microscopy & ct-scan files while they were not BIDS compliant. It is a copy of the pybids config file with the addition of "custom" field needed for `micr` & `ct`.

With this PR, we won't need the "custom" pybids config anymore for `micr`, but will still need it for `ct`.
As such, the only difference between this custom file and the official pybids config  from [here](https://github.com/bids-standard/pybids/blob/master/bids/layout/config/bids.json) is the addition of `ct` on this line.